src/assets/i18n/en.json

```diff
@@ -19,5 +19,18 @@
     "pinboardEmpty": "Your pinboard is empty.",
     "filteredPinboardEmpty": "No pin found for selected types.",
     "filterPins": "Filter pins",
-    "more": "more"
+    "more": "more",
+    "BIBLIOGRAPHY": {
```

Comment:
please use lowerCase keys: just follow the rules you can find in the README in `### 4.2 - Localization` section! 
Also, avoid key nesting: some localized label can be useful somewhere else, thus they should not be connected to bibliography.